sale_automatic_workflow/models/automatic_workflow_job.py

```diff
@@ -116,6 +116,58 @@ def _validate_invoices(self, validate_invoice_filter):
                     invoice.with_company(invoice.company_id), validate_invoice_filter
                 )
 
+    def _do_send_invoice(self, invoice, domain_filter):
+        """Validate an invoice, filter ensure no duplication"""
+        if not self.env["account.move"].search_count(
+            [("id", "=", invoice.id)] + domain_filter
+        ):
+            return "{} {} job bypassed".format(invoice.display_name, invoice)
+
+        # take the context from the actual action_invoice_sent method
+        action = invoice.action_invoice_sent()
+        action_context = action["context"]
+
+        if self.env.su:
+            # sending mail in sudo was meant for it being sent from superuser
+            self = self.with_user(SUPERUSER_ID)
```

Comment:
I'm not sure to understand the goal here, because AFAIU if condition is validated, the user must already be the superuser, or am I missing something?

Reply:
I removed this, too. Thanks!